Tests/GRPCProtobufCodeGenTests/ProtobufCodeGenParserTests.swift

```diff
@@ -65,6 +80,11 @@ final class ProtobufCodeGenParserTests: XCTestCase {
       """
     )
 
+    XCTAssertEqual(parsedCodeGenRequest.dependencies.count, 3)
+    let expectedDependencyNames = ["DifferentModule", "ExtraModule", "GRPCProtobuf"]
+    let parsedDependencyNames = parsedCodeGenRequest.dependencies.map { $0.module }.sorted()
```

Comment:
Is the ordering of dependencies stable? I.e. if we run the code generator multiple times will the order always be the same?

Reply:
That is a good point, I should order them alphabetically before generating. I will add this

Reply:
That's only necessary if the order of `protoToModuleMappings.neededModules` is unstable.

Reply:
neededModules returns the modules imported based on the '.proto' files in alphabetical order, so the ordering is stable. My question is if we should order all of the dependencies (given that we also add "GRPCCore", "GRPCSwiftProtobuf" and the extra imports that are passed in as an option to protoc, besides the one we get from neededModules)

Reply:
Well we insert the GRPC modules into known locations so their ordering is stable. The extra modules _could_ be passed in a different order on different invocations of `protoc-gen-grpc-swift` so we could sort those too. I don't mind whether we do that or just sort the whole lot, the result is the same.